json-resume/src/App.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import './App.css';
+import DataEntry from './DataEntry';
+import Resume from './Resume';
+
+import resume from './resume.json';
+
+import {
```

Comment:
Not sure if you actually need a router/switch here since you have a single page for viewing the resume, you might consider just rendering everything up to App.js instead